### ISSUE TITLE
[new feature request] open explorer in current window rather than left, right or tab.

### ISSUE THREAD

**Issue by fgheng** [weirongxu/coc-explorer#87]
i want to open coc-explorer in current window, would you like to add this feature, thank you very much

**Comment by fgheng** [comment]
or support float window, such as open coc-explorer in the left window which is float in the left.
It's like an android app sliding from the left to open a menu.

**Comment by fgheng** [comment]
> or support float window, such as open coc-explorer in the left window which is float in the left.
> It's like an android app sliding from the left to open a menu.

haha, i feel it is very cool

**Comment by weirongxu** [comment]
> or support float window, such as open coc-explorer in the left window which is float in the left.
> It's like an android app sliding from the left to open a menu.

Indeed, but this already be considered in the floating feature (#51), should support displaying floating windows in different positions
In addition, I'm working on floating feature, and I'll implement this feature in version 0.3.0 and no longer use `conceal` for rendering

**Comment by fgheng** [comment]
thank you very much

**Comment by npearson72** [comment]
Is this a duplicate of https://github.com/weirongxu/coc-explorer/issues/179 or something different? Just trying to understand.
